### PR TITLE
[4.0] Removal of some parameters - PART 1

### PR DESCRIPTION
**Pull request by C-Lodder**
Pull Request for Issue #17072

### Summary of Changes

This is part 1, which removes some parameters from the admin modules:

- `mod_version` (all parameters)
- `mod_feed` (Title and Word Count)

In addition to that, it has some very minor tweaks

### CODE REVIEW

**Comment by C-Lodder** [comment]
RTL option added back into `mod_feed` as discussed with @brianteeman

**Comment by wilsonge** [comment]
You haven't added back the RTL option

**Comment by C-Lodder** [comment]
@wilsonge have now

File: administrator/language/en-GB/en-GB.mod_feed.ini [hunk 1]
```diff
@@ -15,12 +15,8 @@ MOD_FEED_FIELD_ITEMDESCRIPTION_DESC="Show the Description or Intro text of indiv
 MOD_FEED_FIELD_ITEMDESCRIPTION_LABEL="Item Description"
 MOD_FEED_FIELD_ITEMS_DESC="Enter number of RSS items to display."
 MOD_FEED_FIELD_ITEMS_LABEL="Items"
-MOD_FEED_FIELD_RSSTITLE_DESC="Display news feed title."
-MOD_FEED_FIELD_RSSTITLE_LABEL="Feed Title"
 MOD_FEED_FIELD_RSSURL_DESC="Enter the URL of the RSS/RDF/ATOM feed."
 MOD_FEED_FIELD_RSSURL_LABEL="Feed URL"
-MOD_FEED_FIELD_RTL_DESC="Display feed in RTL direction."
+MOD_FEED_FIELD_RTL_DESC="Display feed in RTL direction."
```
Comment (zero-24) [inline administrator/language/en-GB/en-GB.mod_feed.ini]: it looks like there are just extra tabs in this line?

Comment (C-Lodder) [reply administrator/language/en-GB/en-GB.mod_feed.ini]: @zero-24 thanks, fixed.

**Comment by C-Lodder** [comment]
@wilsonge @brianteeman all happy with this?

File: administrator/modules/mod_menu/mod_menu.xml [hunk 2]
```diff
@@ -39,7 +39,7 @@
 					type="radio"
 					label="MOD_MENU_FIELD_CHECK_LABEL"
```
Comment (brianteeman) [inline administrator/modules/mod_menu/mod_menu.xml:40]: you cant remove this one

File: administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini [hunk 3]
```diff
@@ -1,8 +0,0 @@
-; Joomla! Project
```
Comment (brianteeman) [inline administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini:1]: I "think" every module needs at least this *.sys.ini language file

File: administrator/modules/mod_feed/tmpl/default.php [hunk 4]
```diff
@@ -93,15 +99,15 @@
...
+						<h5 class="feed-link"><?php echo $feed[$i]->title; ?></h5>
 					<?php  endif; ?>
 
 					<?php if ($params->get('rssitemdesc') && !empty($text)) : ?>
 						<div class="feed-item-description">
 						<?php
 							// Strip the images.
 							$text = JFilterOutput::stripImages($text);
-							$text = JHtml::_('string.truncate', $text, $params->get('word_count'), true, false);
+							$text = JHtml::_('string.truncate', $text, 0, true, false);
```
Comment (brianteeman) [inline administrator/modules/mod_feed/tmpl/default.php]: is this needed if we dont have a word_count ? does it even do anything

Comment (C-Lodder) [reply administrator/modules/mod_feed/tmpl/default.php]: Yes, cause it strips the HTML (`<p class="lead">`) resulting in much larger text too.

Comment (brianteeman) [reply administrator/modules/mod_feed/tmpl/default.php]: so can you add a comment - something like

// strip html

please

File: administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini (hunk 3 above)
Comment (C-Lodder) [reply administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini:1]: Removing these 2 language files as they exist in `administrator\language\en-GB`

File: administrator/modules/mod_feed/tmpl/default.php (hunk 4 above)
Comment (C-Lodder) [reply administrator/modules/mod_feed/tmpl/default.php]: Done. Used native PHP instead

Comment (brianteeman) [reply administrator/modules/mod_feed/tmpl/default.php]: or at lease make it clear that its not related to the strip images comment

File: administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini (hunk 3 above)
Comment (brianteeman) [reply administrator/modules/mod_version/language/en-GB/en-GB.mod_version.sys.ini:1]: ah ok - didnt spot that - sorry

File: administrator/modules/mod_menu/mod_menu.xml (hunk 2 above)
Comment (C-Lodder) [reply administrator/modules/mod_menu/mod_menu.xml:40]: I'm not removing anything here. Just a class change a couple of lines below

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on f34bdf34acb5d38fb47f35d5c9ec3fc9a244b133<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17075">issues.joomla.org/tracker/joomla-cms/17075</a>.</sub>

**Pull request merged**